adapter/serviceControl/serviceController.go

```diff
@@ -0,0 +1,197 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package serviceControl
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/pborman/uuid"
+	servicecontrol "google.golang.org/api/servicecontrol/v1"
+
+	"istio.io/mixer/adapter/serviceControl/config"
+	"istio.io/mixer/pkg/adapter"
+)
+
+type (
+	builder struct {
+		adapter.DefaultBuilder
+
+		client
+	}
+
+	aspect struct {
+		serviceName string
+		service     *servicecontrol.Service
+		logger      adapter.Logger
+	}
+)
+
+var (
+	name = "serviceControl"
+)
+
+// Register records the builders exposed by this adapter.
+func Register(r adapter.Registrar) {
+	r.RegisterMetricsBuilder(
+		&builder{adapter.NewDefaultBuilder(
+			name,
+			"Push metrics to GCP service controller",
+			new(config.Params),
+		), new(clientImpl)})
+
+	r.RegisterApplicationLogsBuilder(
```

Comment:
service control doesn't have a place for access logs. also, @geeknoid told me that he will merge access log into application log. I will take care of it until that happens.